cmd/run/run_test.go

```diff
@@ -378,6 +379,43 @@ func TestBuildServiceWithPresharedKeyAuthentication(t *testing.T) {
 	}
 }
 
+func TestBuildServiceWithTailSpanExporterEnabled(t *testing.T) {
```

Comment:
I don't think we need this test anymore, right?

Reply:
This test is still useful, but the name is wrong